ui/webui/resources/br_elements/br_toolbar/BUILD.gn

```diff
@@ -12,5 +12,5 @@ polymer_modulizer("br_toolbar_search_field") {
   js_file = "br_toolbar_search_field.js"
   html_file = "br_toolbar_search_field.html"
   html_type = "dom-module"
-  auto_imports = [ "chrome://resources/cr_elements/cr_search_field/cr_search_field_behavior.html|CrSearchFieldBehavior" ]
+  auto_imports = [ "ui/webui/resources/cr_elements/cr_search_field/cr_search_field_behavior.html|CrSearchFieldBehavior" ]
```

Comment:
cc @petemill in case this changes anything for us as far as what we can do

Reply:
definitely LGTM; we tried relative path originally with C80, but for some reason it didn't work. Glad you got it sorted, @mkarolin 😄